test/fixtures/gomod-small/expected-tree.json

```diff
@@ -19,61 +19,61 @@
         },
         "github.com/gdamore/tcell": {
             "name": "github.com/gdamore/tcell",
-            "version": "v1.1.2",
+            "version": "1.1.2",
```

Comment:
🤔 why did this bump have an impact on the version?

Reply:
Because of this https://github.com/snyk/snyk-go-parser/pull/15